dockstore-webservice/src/main/java/io/dockstore/webservice/languages/LanguageHandlerInterface.java

```diff
@@ -144,6 +145,11 @@ public interface LanguageHandlerInterface {
     Map<String, SourceFile> processImports(String repositoryId, String content, Version version,
         SourceCodeRepoInterface sourceCodeRepoInterface, String filepath);
 
+    default Map<String, SourceFile> processOtherFiles(String repositoryId, List<String> otherFilePaths, Version version, SourceCodeRepoInterface sourceCodeRepoInterface, Map<String, SourceFile> alreadyProcessedFileMap) {
```

Comment:
Maybe `processUserSpecifiedFiles` or something like that?  Makes it clearer what they are, but without introducing the notion of `[.dockstore.]yml` to `LanguageHandler`, which is probably good, because they might be specified via a different registration mechanism someday...

Reply:
`User` is better. It at least highlights that `imports` is language-based, whereas these files came from the user. But then all files come from the user some way or another... Naming is hard